fix(ui): view header and og about page

Risyandi opened the pull request:
**Description:**
- view header and og about page

**Changes:**
- adding asset for header
- adding asset for og
- create routes to v2

**Related Issue:**
Closed for the issue link below

close https://github.com/holdex/marketing/issues/348

Risyandi commented:
Hi @angelicawill @williamrusdyputra, could you please review the PR? Thank you.

Risyandi commented:
Hi @angelicawill. 
Yes, correct, this is for about pages. And actually, I am only adding mixins for `section-hero-pages` in path /c/ and for pages in the path /apply/ or /c/ i'm not having any changes.

1. Regarding layout shift, I have been optimizing. 
<details>
  <summary>Performance CLS</summary>
<img width="955" height="491" alt="image" src="https://github.com/user-attachments/assets/c9473848-b52e-421a-b5b0-e8ed9f60f19d" />
 </details>
 
 2. I have fixed it in the last commit.

angelicawill commented:
> Yes, correct, this is for about pages. And actually, I am only adding mixins for `section-hero-pages` in path /c/ and for pages in the path /apply/ or /c/ i'm not having any changes.

@Risyandi If this only affects the /about page and there are no other changes, do we actually need the new v2 path for the other pages?

angelicawill commented:
@Risyandi I suggest to keep this PR scoped to files that relate to the /about page only, to respect the problem statement and PR title. If you'd like to handle the other pages, we can open another PR as needed. that way it's easier to keep track of changes and easier to review

Risyandi commented:
> @Risyandi I suggest to keep this PR scoped to files that relate to the /about page only, to respect the problem statement and PR title. If you'd like to handle the other pages, we can open another PR as needed. that way it's easier to keep track of changes and easier to review

@angelicawill based on the suggestion previously. Mark's recommendation to add path v2. trigger by query params `?v2`

Risyandi commented:
> @Risyandi I suggest to keep this PR scoped to files that relate to the /about page only, to respect the problem statement and PR title. If you'd like to handle the other pages, we can open another PR as needed. that way it's easier to keep track of changes and easier to review

Yes, sure. Well noted, if this PR merges, I will continue to the next PR with another branch

angelicawill commented:
@Risyandi Can you elaborate this point? Not sure I understand.

> > @Risyandi I suggest to keep this PR scoped to files that relate to the /about page only, to respect the problem statement and PR title. If you'd like to handle the other pages, we can open another PR as needed. that way it's easier to keep track of changes and easier to review
> 
> @angelicawill based on the suggestion previously. Mark's recommendation to add path v2. trigger by query params `?v2`

Risyandi commented:
Hi @angelicawill, I think the screenshot will answer your question. Why I'm creating routes v2. Hopefully can help you.
<details>
  <summary>Click to expand for more information</summary>
<img width="1072" height="270" alt="image" src="https://github.com/user-attachments/assets/84f897b3-148f-40ac-a961-bf7c78642d57" />

reference link: https://github.com/holdex/holdex-venture-studio/pull/799#pullrequestreview-3278022637
</details>

Risyandi commented:
Hi @angelicawill, 

Actually, I was also confused than you. If the first instruction is like you mention, it makes sense. 
I have updated the code for this PR. I hope this PR will be easier to review.

Thank you for elaborating and clarifying. Lesson learn for me.

Risyandi commented on src/routes/(pages)/about/+page.svelte:
```diff
@@ -4,21 +4,26 @@
 </script>
 
 <MetaTags
   title={message.title}
   description={message.subtitle}
   path={routes.about}
-  imagePath="/og/about.png"
+  imagePath="/og/v2/about.png"
 />
```
done

Risyandi commented on src/lib/components/ArticleHeaderImage/index.svelte:
```diff
@@ -0,0 +1,53 @@
+
+  onMount(async () => {
+    await tick();
+    const result = await tick();
```
Forgot to remove the line. I used this to check the logs of the function. remove in last commit

angelicawill commented:
<img width="2756" height="1308" alt="CleanShot 2025-10-08 at 13 18 58@2x" src="https://github.com/user-attachments/assets/9f78a641-f274-4f38-a65c-7a9438c17d13" />

I'm assuming this is a draft. Please explicitly re-request for review when it's ready @Risyandi

Risyandi commented:
@angelicawill Based on your review, I have been improving and have tried to fix:

1. I have removed all images unused in the headerpages / og directory.
2. For the OG images have a different format/layout, and so for v2, using og in folder v2.
3. The PR title has been renamed to feat.
4. The breadcrumbs element has been centered for v2.
5. I have added a line break divide element.
6. I have improved and tried the frontend to be pixel-perfect.

Thank you.

angelicawill commented on src/routes/(pages)/index-mixins.pug:
This mixin should be moved to common-mixins as it's needed by multiple pages beyond the index page

angelicawill commented on src/routes/(pages)/common-mixins.pug:
```diff
@@ -2,11 +2,20 @@ mixin content-wrapper
+      class="flex flex-col gap-7.25 xs:gap-4 items-center justify-center mb-7.25 xs:mb-4 smDown:justify-start smDown:items-start max-w-[1028px] tb:max-w-[794px] smDown:max-w-[604px] xs:px-4 w-full"
+    )&attributes(attributes)
+      block
+    +else
```
Why is this needed? You've added the exact same classes for v2 and the previous one. Why do we need a conditional statement for v2?

angelicawill reviewed (changes requested):
1. You have used the wrong OG image for the new v2 page. Please check mark's previous review. https://github.com/holdex/holdex-venture-studio/pull/799#pullrequestreview-3284773568
<img width="1774" height="684" alt="CleanShot 2025-10-08 at 16 00 32@2x" src="https://github.com/user-attachments/assets/bc58d12e-6e28-40a0-8606-558eedd4096c" />

2. Your changes have broken the UI on other pages:
 https://holdex-venture-studio-git-fix-web-hea-a8979f-holdex-accelerator.vercel.app/portfolio, https://holdex-venture-studio-git-fix-web-hea-a8979f-holdex-accelerator.vercel.app/for-startups

Before submitting, please test all pages to ensure your changes don't introduce mistakes elsewhere on the site. Re-check all pages before marking a PR as ready for review

Risyandi commented:
@angelicawill, several points have been checked:
1. have solved
2. The UI is broken because I modified the common mixins `header-wrapper`, adding parameters and the mixins actually used in other pages. So I created the new mixins `header-wrapper-center` because the alignment is centered. 

Thank you.

angelicawill commented on src/routes/(pages)/common-mixins.pug:
```diff
@@ -35,3 +44,21 @@ mixin guideCard(href, title, subtitle, className = "")
     p(class="group-hover:text-accent1-default w-full" textColor="text-t2") #{subtitle}
+
+// hero pages section
+mixin section-hero-pages(imageUrl, effectUrl)
```
`section-hero-image` is a clearer name

angelicawill commented on src/routes/(pages)/common-mixins.pug:
```diff
@@ -2,12 +2,21 @@ mixin content-wrapper
   )&attributes(attributes)
     block
 
+mixin header-wrapper-center
```
`header-wrapper-v2` is a better name so we know what the mixin is used for

angelicawill commented on src/routes/(pages)/about/+page.svelte:
```diff
@@ -4,21 +4,26 @@
   description={message.subtitle}
   path={routes.about}
-  imagePath="/og/about.png"
+  imagePath={'/og/about.png'}
```
OG image is different from what mark mentioned. Please re-read his previous review and try to understand what it means

angelicawill reviewed (changes requested):
<img width="3024" height="1644" alt="CleanShot 2025-10-08 at 21 23 36@2x" src="https://github.com/user-attachments/assets/1cf13d1d-4ae2-40af-8829-65a0892a73de" />

The spacing between the subtitle and divider is not pixel-perfect to the design, it's different
